IDE/Espressif/ESP-IDF/examples/wolfssl_test/main/main.c

```diff
@@ -0,0 +1,166 @@
+/* main.c
+ *
+ * Copyright (C) 2006-2022 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+
+/* ESP-IDF */
+#include <esp_log.h>
+#include "sdkconfig.h"
+
+/* wolfSSL */
+#include <user_settings.h>
+#ifndef WOLFSSL_ESPIDF
+#warning "problem with wolfSSL user settings. Check components/wolfssl/include"
+#endif
+
+#include <wolfcrypt/test/test.h>
+
+/*
+** the wolfssl component can be installed in either:
+**
+**   - the ESP-IDF component directory
+**
+**       ** OR **
+**
+**   - the local project component directory
+**
+** it is not recommended to install in both.
+**
+*/
+
+/*
+** although the wolfcrypt/test includes a default time setting,
+** see the enclosed optional time helper for adding NNTP.
+** be sure to add "time_helper.c" in main/CMakeLists.txt
+*/
+#undef WOLFSSL_USE_TIME_HELPER
+#if defined(WOLFSSL_USE_TIME_HELPER)
```

Comment:
I think this include will never get included. Is line 52 supposed to be commented out in certain cases?

Reply:
That's correct: using the time helper code is optional. It is disabled by default.

Reply:
In a future version, it would be a good idea to implement the ESP32 time as `current_time` such as the [example for micrium](https://github.com/wolfSSL/wolfssl/blob/dfa997a6fd9569ca9bd588e51c56e9580ba07590/wolfcrypt/benchmark/benchmark.c#L8340).